apps/kubernetes-external-secrets/OWNERS

```diff
@@ -1,6 +1,12 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
-    - wg-k8s-infra-leads
+- ameukam
+- spiffxp
```

Comment:
AFAIK @ameukam and I are the only two who have domain expertise on wg-k8s-infra's use of this, I'd rather signal that to the other leads